consensus/src/marshal/config.rs

```diff
@@ -36,25 +36,6 @@ where
     /// Prunable archive partition prefix.
     pub prunable_items_per_section: NonZeroU64,
 
-    /// The number of items to store per section in immutable archives.
-    pub immutable_items_per_section: NonZeroU64,
-
-    /// The initial size of the freezer table.
-    pub freezer_table_initial_size: u32,
-
-    /// The frequency (in number of resizes) at which to check if the freezer table
-    /// should be resized.
-    pub freezer_table_resize_frequency: u8,
-
-    /// The number of items to add to the freezer table when resizing.
-    pub freezer_table_resize_chunk_size: u32,
-
-    /// The target size of the freezer journal.
-    pub freezer_journal_target_size: u64,
-
-    /// The compression level to use for the freezer journal.
-    pub freezer_journal_compression: Option<u8>,
-
     /// The buffer pool to use for the freezer journal.
     pub freezer_journal_buffer_pool: PoolRef,
```

Comment:
I think this should just be `buffer_pool` (if used at all)?

Reply:
Renamed! It's still used by the ephemeral `Cache` in marshal